_release-pypi/action.yml

```diff
@@ -28,7 +28,7 @@ runs:
   steps:
   
     - name: "Set up Python"
-      uses: actions/setup-python@v4
+      uses: pyansys/actions/_setup-python@main
       with:
```

Comment:
Remember that whenever we do a release, we should change all these "@main" tags in the release branch to point to its corresponding release :)

Reply:
lol, didn't know there was a guy named "main" hahaha!

Reply:
That's right! I noticed this too while working with this. Let's keep that in mind.